tests/testsuite/artifact_dep.rs

```diff
@@ -1603,6 +1603,7 @@ fn dep_of_artifact_dep_same_target_specified() {
 
                     [dependencies]
                     bar = {{ path = "bar", artifact = "bin", target = "{target}" }}
+                    one = {{ path = "one", artifact = "bin", target = "x86_64-fortanix-unknown-sgx" }}
```

Comment:
Is there some way to test for _another_ target besides the one already represented by `target` on line 1605? I know this will fail in Github Actions because the OS images don't have the fortanix platform installed.

Reply:
Is https://docs.rs/cargo-test-support/latest/cargo_test_support/cross_compile/fn.alternate.html what you are looking for?

Reply:
Actually, I don't think this extra package in the project is necessary. I added it during development to see more helpful print output.